Frontend/ui-customer/src/Screens/Types.purs

```diff
@@ -434,6 +434,7 @@ data Stage = HomeScreen
            | ChatWithDriver
            | FindEstimateAndSearch
            | RetryFindingQuote
+           | InitialStage
```

Comment:
why this stage is required?

Reply:
this stage is used to know that we just started app and haven't called any API